web/pages/Chub/Chub.tsx

```diff
@@ -0,0 +1,62 @@
+import { Component, createMemo, createSignal } from 'solid-js'
+import PageHeader from '../../shared/PageHeader'
+import { setComponentPageTitle } from '../../shared/util'
+import Tabs from '../../shared/Tabs'
+import FilterSettings from './FilterSettings'
+import CharList from './CharList'
+import { ChubOptions } from '../../store/chub'
+import BookList from './BookList'
+
+export const chubOptions: ChubOptions = {
```

Comment:
This should be in state (if it's used in multiple places) or a signal if it's used in a single place.
I believe we've used local storage elsewhere to cache/re-hydrate these sorts of sort/filtering values.